.pre-commit-config.yaml

```diff
@@ -4,3 +4,15 @@ repos:
     rev: v2.0.0
     hooks:
```

Comment:
We would also need to remove this, i will take up in some other PR, 
This looks good, Thanks for contributing 👍